src/iotjs_module_buffer.cpp

```diff
@@ -82,6 +82,14 @@ size_t BufferWrap::length() {
 }
 
 
+static size_t boundRange(int index, size_t low, size_t upper)
+{
+  if (index < static_cast<int>(low)) return low;
+  if (index > static_cast<int>(upper)) return upper;
```

Comment:
I'm not sure what is the correct style, but this function looks odd to me. I think braces must be used in if statements and the left brace at the start of the function mustn't be on a separate line.